packages/server/lib/mud/getNetworkConfig.ts

```diff
@@ -11,7 +11,7 @@ type NetworkConfig = SetupContractConfig & {
 };
 
 export async function getNetworkConfig(): Promise<NetworkConfig> {
-    const params = new URLSearchParams(window.location.search);
+    const params = new URLSearchParams(typeof window === "undefined" ? "" : window.location.search);
```

Comment:
could also just remove all references to `params` if this is intended to only run in node